Condition expressions/Single-line if-else statement/task.md

```diff
@@ -1,6 +1,9 @@
-## Single-line if-else statement
+As you saw in the previous tasks, a simple `if-else` statement can in fact be written as a single line.
 
-As you saw in the previous tasks, a simple `if-else` statement can in fact be written as a single line. Rewrite the statement in the code editor so that it
-fits in one line but the results of its execution remain exactly the same.
+### Task
+Rewrite the statement in the code editor so that it
+fits in one line but the results of its execution remain exactly the same:
+- If `my_random_number` is greater than 50, program should print `my_random_number` itself.
```

Comment:
If `my_random_number` is greater than 50, the program should print the value of `my_random_number`.